install/kubernetes/helm/istio/files/injection-template.yaml

```diff
@@ -9,7 +9,8 @@ initContainers:
 {{- else }}
   image: "{{ .Values.global.hub }}/{{ .Values.global.proxy_init.image }}:{{ .Values.global.tag }}"
 {{- end }}
-  args:
+  command:
+  - istio-iptables
```

Comment:
I don't remember exactly - but for 'distroless' ( without a shell ) I thought it's a special format, and includes full path ?

Reply:
Distroless still has `$PATH` I think, this seemed to work fine, and we have distroless tests